backend/tarsy/controllers/chat_controller.py

```
@@ -0,0 +1,477 @@
+    - stage_execution_id links to StageExecution for tracking assistant response
+    
+    **WebSocket Events:**
+    - User message triggers ChatUserMessageEvent
+    - AI response streams via existing LLMStreamChunkEvent events
+    """,
+)
+async def send_message(
+    request: Request,
+    chat_id: str = Path(..., description="Chat identifier"),
+    message_request: ChatMessageRequest = None,
+    chat_service: Annotated[ChatService, Depends(get_chat_service)] = None,
```

Ruff (0.14.3)

Unused function argument: `chat_service`

(ARG001)